### PR TITLE
[2-4] login page creation

### PR DESCRIPTION
**Pull request by HenadzV**
Login page created
Added main guard
Added login guard

### CODE REVIEW

File: exalearn-ui/src/app/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
```
Comment (YanaBr) [inline exalearn-ui/src/app/auth.service.ts]: As I see, service is not inside 'services' folder. Please, move it if I right

File: exalearn-ui/src/app/guards/auth.guard.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@Injectable({ providedIn: 'root' })
+export class AuthGuard implements CanActivate {
+	constructor(private authService: AuthService, private router: Router) {}
+
+	canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> | boolean {
+		this.authService.isAuthenticated().then((isAuth) => {
+			if (isAuth) {
+				this.router.navigateByUrl('/main');
+			}
+			// else {
```
Comment (YanaBr) [inline exalearn-ui/src/app/guards/auth.guard.ts]: Remove comment, please

File: exalearn-ui/src/app/auth.service.ts (hunk 1 above)
Comment (HenadzV) [reply exalearn-ui/src/app/auth.service.ts]: Moved to service directory

File: exalearn-ui/src/app/guards/auth.guard.ts (hunk 2 above)
Comment (HenadzV) [reply exalearn-ui/src/app/guards/auth.guard.ts]: Comment removed

**Review by YanaBr — APPROVED** [review]

**Review by KuznetsovR — APPROVED** [review]

**Review by HrebenYury — APPROVED** [review]

**Review by KotdenGit — APPROVED** [review]

**Pull request merged**